### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by janpc**

### CODE REVIEW

File: packages/web/src/components/EditUserForm/EditUserForm.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+  const formData = new FormData();
+
+  useEffect(() => {
+    if (currentUser.birthday) {
+      setUser({ ...currentUser, birthday: currentUser.birthday.slice(0, 10) });
+    } else {
+      setUser(currentUser);
+    }
+  }, []);
```
Comment (soyyogi) [inline packages/web/src/components/EditUserForm/EditUserForm.js:33]: git suggests useEffect need currentUser as a dependency or remove dependency array.

File: packages/api/src/routes/user-routes.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const Router = require("express").Router;
+var multer = require("multer");
+var upload = multer({ dest: "uploads/" });
```
Comment (soyyogi) [inline packages/api/src/routes/user-routes.js:3]: When files are added to "uploads" directory and once they have been uploaded to couldinary, we must remove them from "uploads" directory.

**Review by soyyogi — CHANGES_REQUESTED** [review]
I have provided two comments for changes. Moreover I do not have much knowledge of styled components, So I didn't review that part.

**Comment by VictorGreco** [comment]
retest this please

**Review by soyyogi — APPROVED** [review]

**Review by soyyogi — CHANGES_REQUESTED** [review]
Please run yarn run lint:format

**Review by soyyogi — APPROVED** [review]

**Pull request merged**